[WIP] v6 Docs Dev (jschalz) (#2025)
loft-sh/devspace: docs/pages/configuration/images/image-definition/tags.mdx

    tags:
    # The five # are replaced with 5 random characters. The variable ${devspace.git.branch} is replaced
    # with the current branch the user is on
    - 'dev-${devspace.git.branch}-#####'
```

:::info Unique Tags For Development

> **Make sure tags are unique** when defining a custom tagging schema for development. Unique tags ensure that your application gets started with the newly built image instead of using an older, cached version. Therefore, it is highly recommended for non-production tags to either use `#` placeholders or `devspace.timestamp` as a suffix in your tagging schema (see [example below](#example-custom-tagging-schema)).

Reviewer: ```suggestion
**Make sure tags are unique** when defining a custom tagging schema for development. Unique tags ensure your application starts with the newly built image instead of using an older, cached version. Therefore, it is highly recommended for non-production tags to either use `#` placeholders or `devspace.timestamp` as a suffix in your tagging schema (see [example below](#example-custom-tagging-schema)).
```

images:
  backend:
    image: john/appbackend
    tags:
    - latest
    - dev-backend-${devspace.git.commit}-#####
```
**Explanation:**

> The above example would always use the tag `latest` and additionaly generate random tags which could look like this one: `dev-backend-b6caf8a-Jak9i`. This example tag consists of the following substrings defined by the tagging schema:

Reviewer: ```suggestion
The above example always uses the tag `latest`, while generating random tags which could look like this one: `dev-backend-b6caf8a-Jak9i`. This example tag consists of the following substrings defined by the tagging schema:
```
